Primo Normalized XML.js

```diff
@@ -342,9 +342,11 @@ function stripAuthor(str) {
 	// e.g. Wheaton, Barbara Ketcham [former owner]$$QWheaton, Barbara Ketcham
 	str = str.replace(/^(.*)\$\$Q(.*)$/, "$2");
 	return str
-		// Remove year
-		.replace(/\s*,?\s*\(?\d{4}-?(\d{4})?\)?/g, '')
-		// Remove things like (illustrator). TODO: use this to assign creator type?
+	// Remove year
+		.replace(/\s*,?\s*\(?\d{4}-?(\d{4}|.{3})?\)?/g, '')
```

Comment:
do you mean to use the wildcard `.` there? Otherwise this should be `\.{3]`

Reply:
No, I mean specifically for dots (author like '1962-...')

Reply:
And it's always three literal dots, an actual ellipsis?

Reply:
Yes, this specific pattern

Reply:
In that case, it should be escaped, as @adam3smith mentioned: `\.{3}`. This current pattern will match any three characters.